app/views/sources/_image.html.erb

```diff
@@ -1,20 +1,15 @@
+<% content_for :head_script do %>
+<% javascript_include_tag 'openseadragon' %>
+<% end %>
```

Comment:
Same as above:  Could we put this in :foot_script instead? This will keep it consistent with changes in PR#94.